.rubocop.yml

```diff
@@ -120,3 +120,11 @@ Style/MethodName:
 Style/VariableName:
   Include:
     - 'src/lib/**/*.rb' # force known variables only for new code
+
+# alias method is safer option
+Style/Alias:
+  EnforcedStyle: prefer_alias_method
+
+# performance gain is not so big and readability is better
+Performance/RedundantBlockCall:
+  Enabled: false
```

Comment:
I'd like to add this globally, see https://github.com/yast/yast-devtools/pull/107/files#diff-71e6a5f24eae84627de24615bb8d6545R218

So maybe we should discuss and merge https://github.com/yast/yast-devtools/pull/107 first...